src/current/_includes/releases/v23.1/v23.1.25.md

```diff
@@ -0,0 +1,80 @@
+    - String that can be parsed as a valid JSON array of issuer URLs list: `['example.com/adfs','https://accounts.google.com']`.
+    - String that can be parsed as a valid JSON and deserialized into a map of issuer URLs to corresponding JWKS URIs. In the third case, CockroachDB will override the JWKS URI present in the issuer's well-known endpoint: `'{ "issuer_jwks_map": { "https://accounts.google.com": "https://www.googleapis.com/oauth2/v3/certs", "example.com/adfs": "https://example.com/adfs/discovery/keys" } }'`.
+
+    When `issuer_jwks_map` is set, CockroachDB directly uses the JWKS URI to get the key set. In all other cases where [`server.jwt_authentication.jwks_auto_fetch.enabled`]({% link v23.1/sso-sql.md %}#cluster-settings) is set CockroachDB obtains the JWKS URI first from issuer's well-known endpoint and then uses this endpoint. [#128751][#128751]
+
+<h3 id="v23-1-25-sql-language-changes">SQL language changes</h3>
+
+- Added the `sql.auth.grant_option_inheritance.enabled` [cluster setting]({% link v23.1/cluster-settings.md %}). The default value is `true`, which results in behavior that matches the existing behavior of CockroachDB. When set to `false`, the [`GRANT OPTION`]({% link v23.1/grant.md %}) is not inherited via role membership, which will prevent the descendant roles from being able to grant the privilege to others. The privilege itself is still inherited via role membership. [#126862][#126862]
+- Added the `sql.auth.grant_option_for_owner.enabled` [cluster setting]({% link v23.1/cluster-settings.md %}). The default value is `true`, which results in behavior that matches the existing behavior of CockroachDB. When set to `false`, then the [`GRANT OPTION`]({% link v23.1/grant.md %}) is not implcitly given to the owner of an object. The object owner still implicitly has all privileges on the object, just not the ability to grant them to other users. [#127005][#127005]
+
+<h3 id="v23-1-25-db-console-changes">DB Console changes</h3>
+
+- The [Database]({% link v23.1/ui-databases-page.md %}) details and Table details pages in the [DB Console]({% link v23.1/ui-overview.md %}) now display the correct `"stats last created"` value. [#126415][#126415]
+- The [Database]({% link v23.1/ui-databases-page.md %}) and Tables pages in the [DB Console]({% link v23.1/ui-overview.md %}) will show a loading state while loading information for databases and tables including size and range counts. [#127699][#127699]
+- In the [Database]({% link v23.1/ui-databases-page.md %}) details page, the table name will no longer appear with quotes around the schema and table name. [#127763][#127763]
+- [`ALTER ROLE`]({% link v23.1/alter-role.md %}) events in the [DB Console]({% link v23.1/ui-overview.md %}) event log now render correctly when the event does not contain any role options. [#126563][#126563]
+- Fixed a bug where the [Databases]({% link v24.1/ui-databases-page.md %}) page crashed if the range information was not available. [#127090][#127090]
```

Comment:
"range" could link to https://www.cockroachlabs.com/docs/v24.2/architecture/glossary#architecture-range